firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -821,19 +815,12 @@ static void check_rx_timeouts( void )
 // the setup routine runs once when you press reset:
 void setup( void ) 
 {
-
-    // duty Scalers good for 0x05
-    //SLADutyMax = 50;
-    //SLADutyMin = 5;
-    //SLRDutyMax = 50;
-    //SLRDutyMin = 20;
-
-
     // duty Scalers good for 0x02
```

Comment:
What does this mean? It would be good to elaborate on this comment and what the 0x02 is in reference to.